SecretSanta/test/SecretSanta.Business.Tests/UserTests.cs

```diff
@@ -0,0 +1,82 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace SecretSanta.Business.Tests
+{
+    [TestClass]
+    public class UserTests
+    {
+        const int Id = 7;
+        const string FirstName = "Sean";
+        const string LastName = "Scura";
+
+        [TestMethod]
+        [DataRow(nameof(User.FirstName), FirstName)]
+        [DataRow(nameof(User.LastName), LastName)]
+        [DataRow(nameof(User.Id), Id)]
+        public void UserObject_CanPassValueToProperties_AcceptsValues(string propertyName, object value)
+        {
+            List<Gift> gifts = new List<Gift>();
+            User user = new User(Id, FirstName, LastName, gifts);
+
+            Assert.IsNotNull(user);
+            Assert.AreEqual(value, GetProperty(propertyName));
+            Assert.IsNotNull(user.Gifts);
+        }
+
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void UserObject_NullGifts_ThrowsException()
+        {
+            List<Gift> gifts = null!;
+            User user = new User(Id, FirstName, LastName, gifts);
```

Comment:
In cases where you don't need the value you can use a throw away variable.
```suggestion
            var _ = new User(Id, FirstName, LastName, gifts);
```